lib/std/start.zig

```diff
@@ -25,6 +25,12 @@ pub const simplified_logic =
     builtin.cpu.arch == .spirv32 or
     builtin.cpu.arch == .spirv64;
 
+pub const StackSizeStrategy = enum {
```

Comment:
You might add the comments from your commit message as doc comments on the enum here.